src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+$gap : 101px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  padding: 10px;
+  margin: auto;
+  gap: 1px;
+  width: 726px;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    background-color: #eee;
+    border: #000 solid 1px;
+    width: 100px;
+    height: 100px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    color: #000;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    transition: transform 0.5s ease;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      cursor: pointer;
+    }
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day--#{$i}::before {
+    content: "#{$i}";
+  }
+}
+
+$start-days:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
```

Comment:
I recommend moving the variables to a separate file.

Reply:
done